drivers/barometer.hpp

```diff
@@ -131,7 +131,7 @@ class Barometer
      *
      * \return  Altitude (global frame)
      */
-    static float altitude_from_pressure(float pressure, float pressure_at_sea_level = compute_pressure_at_sea_level());
+    static float compute_altitude_from_pressure(float pressure, float pressure_at_sea_level = pressure_at_sea_level());
```

Comment:
with this line, you have again the compilation issue in clang...
Apparently, you can't have the name of a function and a variable in the same function definition